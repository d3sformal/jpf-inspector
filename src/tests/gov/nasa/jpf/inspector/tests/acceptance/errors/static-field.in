cr bp method=main
run && wait
print arrayOfIntegers
print thisIsNotAStaticThing
del bp 1