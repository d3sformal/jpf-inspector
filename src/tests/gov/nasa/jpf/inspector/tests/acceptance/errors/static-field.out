INFO: JPF created and connected, SuT is started
INFO: Breakpoint hit: [1 :  state=ENABLED hits=1 hitsTotal=1 method = main]
	In method "main".
arrayOfIntegers (int[]) = null
ERR: No entry (field, local variable, parameter) with the name "thisIsNotAStaticThing" exists (maybe we are inside a native method).
Breakpoint with ID 1 successfully deleted.
INFO: JPF Terminating